api/models.py

```diff
@@ -247,6 +247,13 @@ class Node(DatabaseModel):
     holdoff: Optional[datetime] = Field(
         description="Node expiry timestamp while in Available state"
     )
+    user: Optional[str] = Field(
```

Comment:
Maybe `owner` would be a bit better, but that's a detail.  We can always tweak things like this a bit later.

Reply:
I agree. I will rename it.

Reply:
Done.